### PR TITLE
Release 4.3.0

### PR DESCRIPTION
**Pull request by ashie**

### CODE REVIEW

**Comment by ashie** [comment]
We'll release fluentd 1.14.3 before releasing td-agent 4.3.0 to merge following in_tail fixes:

* https://github.com/fluent/fluentd/pull/3541
* https://github.com/fluent/fluentd/pull/3542

**Comment by cosmo0920** [comment]
FYI: chef is already released win32-service v2.3.0 or later:
https://github.com/chef/win32-service/commit/bde5569a9512f2f7ae07a754ea29f4c7b7b1abd0

**Comment by cosmo0920** [comment]
Now win32-service gem v2.3.2 is out in rubygems!
https://rubygems.org/gems/win32-service/versions/2.3.2

**Comment by ashie** [comment]
TODO: Cannot upgrade td due to https://github.com/treasure-data/td/pull/252

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,65 @@
 # Treasure Agent 4 changelog
 
+## Release v4.3.0 - 2021/11/30
```
Comment (ashie) [inline CHANGELOG.md:3]: The release date will be replaced with actual released date later.

**Comment by ashie** [comment]
We'll publish release announcement for this release at https://www.fluentd.org/blog/
In this announcement, we'll also announce drop schedule:

* td-agent 3 is no longer supported, it uses Ruby 2.4 and it's already EOL
* td-agent 4 for xenial is no longer supported, because [standard support of xenial is already ended](https://wiki.ubuntu.com/Releases).

We'll recommend [enterprise support](https://www.fluentd.org/enterprise_services) for such old versions if users request it.

Do you have any comments? @cosmo0920 @repeatedly

**Comment by cosmo0920** [comment]
I have no concerns for this comment.

**Pull request merged**